DynamoDB/StandaloneFSharp/scratchpad.fsx

```diff
@@ -0,0 +1,67 @@
+﻿#r @".\packages\AWSSDK.Core.3.1.2.1\lib\net45\AWSSDK.Core.dll"
+#r @".\packages\AWSSDK.DynamoDBv2.3.1.1.1\lib\net45\AWSSDK.DynamoDBv2.dll"
+#r @".\packages\FSharp.Control.AsyncSeq.2.0.1\lib\net45\FSharp.Control.AsyncSeq.dll"
+
+#load "dynamoDb.fs"
+#load "dynamoEventStore.fs"
+
+open Amazon.DynamoDBv2
+open Amazon.DynamoDBv2.DocumentModel
+open Amazon.DynamoDBv2.Model
+open Amazon.Runtime
+open RealtyShares
+open RealtyShares.DynamoDb
+open RealtyShares.DynamoEventStore
+
+let awsAccessKey = ""
+let awsSecretKey = ""
+let tableName = "scratchpad-table"
+
+async {
```

Comment:
I like keeping a scratchpad.fsx file for interactive programming purposes. The duplicate code isn't necessary, but it's kind of like a quasi unit test case that makes the script easier to jump into.